<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helloQTClass</class>
 <widget class="QMainWindow" name="helloQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>helloQT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menulu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Compilar"/>
    <addaction name="action_Guardar"/>
    <addaction name="action_Salir"/>
   </widget>
   <addaction name="menulu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Compilar">
   <property name="text">
    <string>&amp;Compilar</string>
   </property>
  </action>
  <action name="action_Guardar">
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="helloqt.qrc"/>
 </resources>
 <connections/>
</ui>
